src/com/lftechnology/java/training/krishna/basics/print_day_in_word/PrintDayInWordDemo.java

```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.java.training.krishna.basics.print_day_in_word;
+
+import java.util.logging.Logger;
+
+public class PrintDayInWordDemo {
+	private final static Logger LOGGER = Logger.getLogger(PrintDayInWord.class.getName());
+	public static void main(String[] args) {
+		int day = 5;
+		PrintDayInWord printdayinword = new PrintDayInWord();
```

Comment:
follow the java naming conventions.
have a look at: http://www.oracle.com/technetwork/java/codeconventions-135099.html